backend/class/mentee.php

```diff
@@ -0,0 +1,151 @@
+<?php
+    class Mentee{
+
+        private $conn;
+
+        private $db_table = "users";
+
+        public $id;
+        public $name;
+        public $email;
+        public $password;
+        public $user_level;
+        public $first_login;
+
+        public function __construct($db){
+            $this->conn = $db;
+        }
+
+        public function getMentees(){
+            $sqlQuery = "SELECT id, name, email, password, user_level, first_login FROM " . $this->db_table . "";
+            $stmt = $this->conn->prepare($sqlQuery);
+            $stmt->execute();
+            return $stmt;
+        }
+
+        public function createMentee(){
+            $sqlQuery = "INSERT INTO
+                        ". $this->db_table ."
+                    SET
+                        name = :name, 
+                        email = :email, 
+                        password = :password, 
+                        user_level = :user_level, 
+                        first_login = :first_login,
+                        account_activation = :account_activation";
+        
+            $stmt = $this->conn->prepare($sqlQuery);
+        
+
+            $this->name=htmlspecialchars(strip_tags($this->name));
+            $this->email=htmlspecialchars(strip_tags($this->email));
+            $this->password=password_hash(htmlspecialchars(strip_tags($this->password)), PASSWORD_DEFAULT);
+            $this->user_level=0;
+            $this->first_login=0;
+            $this->account_activation=0;
+        
+
+            $stmt->bindParam(":name", $this->name);
+            $stmt->bindParam(":email", $this->email);
+            $stmt->bindParam(":password", $this->password);
+            $stmt->bindParam(":user_level", $this->user_level);
+            $stmt->bindParam(":first_login", $this->first_login);
+            $stmt->bindParam(":account_activation", $this->account_activation);
+        
+            if($stmt->execute()){
+               return true;
+            }
+            return false;
+        }
+
+        public function firstloginMentee(){
```

Comment:
Actually why is it called first login? 🤔

Reply:
The first login function is responsible for invoking the form needed to complete the information about the variables affecting the calculation of caloric demand.

Reply:
Then it's onboarding, not first login 😉